default-systemd-target.ks.in

```diff
@@ -0,0 +1,18 @@
+#version=DEVEL
+#test name: default-systemd-target
+#
+# Test multi-user.target should be set by default as the default systemd target.
```

Comment:
I think that the default target depends also on the UI mode and some installed packages. Maybe try to look for `SetDefaultTarget` in the code.

Reply:
I've extended the tests to cover GUI modes as well.